objc/A0LockReactModule.m

```diff
@@ -97,10 +97,16 @@ @implementation A0LockReactModule
     });
 }
 
+RCT_EXPORT_METHOD(getDelegationToken:(NSDictionary *)options callback:(RCTResponseSenderBlock)callback) {
```

Comment:
We'll probably redo some parts of auth0.js so I'd not take it into account. I forgot about node-auth0, so let's keep it as it its (I don't like the name but let's keep "consistency")